test/unit/org/apache/cassandra/gms/EndpointStateTest.java

```diff
@@ -114,11 +117,11 @@ public void testMultiThreadWriteConsistency() throws InterruptedException
     /**
      * Test that two threads can update the state map concurrently.
      */
-    private void innerTestMultiThreadWriteConsistency() throws InterruptedException
+    private void innerTestMultiThreadWriteConsistency() throws InterruptedException, UnknownHostException
     {
         final Token token = DatabaseDescriptor.getPartitioner().getRandomToken();
         final List<Token> tokens = Collections.singletonList(token);
-        final String ip = "127.0.0.1";
+        final InetAddress ip = InetAddress.getByName("127.0.0.1");
```

Comment:
sadly this will try to resolve, since this should be loopback its mostly fine, but would be good to use `java.net.InetAddress#getByAddress(java.lang.String, byte[])`

```
InetAddress.getByAddress(null, new byte[] { 127, 0, 0, 1})
```

this throws the same exception, but doesn't try to resolve

Reply:
ack - much better, thanks.